src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/jobs/promptingCom/PromptComListBuilder.kt

```diff
@@ -97,15 +98,14 @@ class PromptComListBuilder(
   fun excludeOutOfRangeDates(cases: List<CaseWithEmailAndStartDate>, startDate: LocalDate, endDate: LocalDate) =
     cases.filter { (_, _, releaseDate) -> releaseDate in startDate..endDate }
 
-  fun buildEmailsToSend(cases: List<CaseWithEmailAndStartDate>): List<Com> {
+  fun buildEmailsToSend(cases: List<CaseWithEmailAndStartDate>): List<PromptComNotification> {
```

Comment:
Swapped from the `Com` to the old `PromptLicenceCreationRequest` swagger resource, which I've moved, removed swagger docs and renamed.. BUT IT'S THE SAME THING RIGHT! 😂